### PR TITLE
add more explanatory text in GPU notebooks

### PR DESCRIPTION
**Pull request by jameslamb**
This PR adds more explanatory text to the Snowflake GPU notebooks, based on the changes in #58 .

### CODE REVIEW

File: examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb [hunk 1]
```diff
@@ -4,9 +4,7 @@
    "cell_type": "markdown",
```
Comment (skirmer) [inline examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb:4]: <p>This header's not rendered for some reason..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Review by skirmer — APPROVED** [review]
Aside from one comment I left, :shipit:

File: examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb (hunk 1 above)
Comment (jameslamb) [reply examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb:4]: oh weird! looking...

Comment (jameslamb) [reply examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb:4]: it's rendering ok for my in my jupyterlab

image

and on GitHub https://github.com/saturncloud/examples/blob/feat/gpu-notebook-flow/examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb

image

So I think we should just conclude / hope it's a bug in ReviewNB

**Pull request merged**